src/main/java/TARS/ui/TARS.java

```diff
@@ -0,0 +1,559 @@
+                        "Invalid Flags: /by . TARS.command.CommandType EVENT only has /from and /to flags.");
+            }
+
+            int fromIndex = commandBody.toLowerCase().indexOf("/from");
+            int toIndex = commandBody.toLowerCase().indexOf("/to");
+
+            if (fromIndex > toIndex) {
+                throw new TARSInvalidCommandBodyException(
+                        "Invalid Flags: flags /from and /to must be in that order."
+                );
+            }
+
+            description = commandBody.substring(0, fromIndex).trim();
+            String fromString = commandBody.substring(fromIndex + 5, toIndex).trim();
+            String toString = commandBody.substring(toIndex + 3).trim();
+
+            if (description.isEmpty()) {
+                throw new TARSInvalidCommandBodyException("Invalid Argument: TARS.command.CommandType EVENT has empty description.");
+            }
+
+            if (fromString.isEmpty() || toString.isEmpty()) {
+                throw new TARSInvalidCommandBodyException("Invalid Argument: TARS.command.CommandType EVENT has empty start/end time.");
+            }
+
+            break;
+        default:
+            break;
+        }
+        return commandBody;
+    }
+
+    /**
+     * Ensures that data directory and data file exists
+     * If the data file or directory do not exist, they are created
+     *
+     * Exception handling: Handles IOExceptions and Security exceptions
+     */
+    private static void checkDataPathExists() {
+        File dataFolder = new File(dataFolderPath);
+        File dataFile = new File(dataFilePath);
+        try {
+            if (!dataFolder.exists()) {
+                dataFolder.mkdir();
+                dataFolder = new File(dataFolderPath);
+            }
+            if (!dataFile.exists()) {
+                dataFile.createNewFile();
+            }
+
+
+        } catch (SecurityException e) {
+            System.out.println("SecurityException: " + e.getMessage());
+        } catch (IOException e) {
+            System.out.println("IOException: " + e.getMessage());
+        }
+    }
+
+    /**
+     * Reads task data from hardcoded file path, processes each line and stores in array
+     *
+     * Exception handling: FileNotFoundException if data file is missing/corrupted
+     * @return array of raw task data strings
+     */
+
+    private static ArrayList<String> loadFile() {
+        ArrayList<String> rawTaskData = new ArrayList<String>();
+        checkDataPathExists();
+        File dataFile = new File(dataFilePath);
+
+        String nextLine;
+
+        try {
+            Scanner fileScanner = new Scanner(dataFile);
+
+            while (fileScanner.hasNextLine()) {
+                nextLine = fileScanner.nextLine().trim();
+                if (nextLine.endsWith(")")) {
+                    nextLine = nextLine.substring(0, nextLine.length() - 1);
+                }
+
+                rawTaskData.add(nextLine);
+            }
```

Comment:
There are ~3 levels of nesting here, refactoring the code to a method will improve readability.